packages/cisco-sdk-node/src/options.ts

```diff
@@ -77,6 +77,20 @@ export function _configDefaultOptions(
       Consts.DEFAULT_PAYLOADS_ENABLED
     );
 
+  const verified_token = verify_token(options.ciscoToken);
+
+  if (!verified_token) {
```

Comment:
Cool usability solution, but there's a lot of complexity - do you think the token source is relevant? I mean if we print token value already